mass-stop-start-cf-apps/README.md

```diff
@@ -0,0 +1,62 @@
+
+SAP Automation Pilot has the capabilities to perform mass operations on tens or even hundreds of Cloud Foundry applications, including stop, start or restart. In addition to this, these operations can be performed regularly with the [Scheduled Executions functionality](https://help.sap.com/docs/AUTOMATION_PILOT/de3900c419f5492a8802274c17e07049/96863a2380d24ba4bab0145bbd78e411.html).
+
+This example includes a total of four commands. *MassStopCfAppsInSpace* and *MassStartCfAppsInSpace* dynamically find and stop/start all applications in the given space. Similarly, *MassStopCfAppsInOrg* and *MassStartCfAppsInOrg* do the same thing but on organization level - the applications across all spaces are collected and then started/stopped. In addition, it's possible to ignore certain spaces and not perform any operations on them.
+
+![Mass Stop Apps in Org](./assets/mass-stop-org.png)
+![Mass Start Apps in Org](./assets/mass-start-org.png)
+
+:warning: The commands within this example might cause downtime for a large amount of applications. Please make sure that you are not using them on productive systems.
+
+## Requirements
+
+To use this example you'll need the following:
+
+* Multiple Cloud Foundry spaces with at least one application in each
+* Technical user with **Space Developer** role in each space
+
+:warning: If you are planning to use SAP Universal ID, please keep [SAP Note 3085908](https://launchpad.support.sap.com/#/notes/3085908) in mind
+
+Check out the [documentation](https://help.sap.com/docs/btp/sap-business-technology-platform/administration-and-operations-in-cloud-foundry-environment) for more information.
+
+## How to use
+
+Import the content of [examples catalog](catalog.json) in your Automation Pilot tenant.
+The examples includes four commands - *MassStopCfAppsInSpace*, *MassStartCfAppsInSpace*, *MassStopCfAppsInOrg* and *MassStartCfAppsInOrg*. Navigate to any of them and to trigger them.
+
+You'll need to provide values for the following input keys:
+
+* *region* - Technical name of your SAP BTP region, e.g. cf-eu10, cf-us20
+* *subAccount* - Name of your Cloud Foundry organization
+* *resourceGroup* - Name of your Cloud Foundry space
+* *user* - Email or ID of your technical user
+* *password* - Password of your technical user
+* *identityProvider* - Optional: origin key of your identity provider. Defaults to sap.ids
+* *excludedSpaces* - Technical names of the Cloud Foundry spaces in which the applications should not be started/stopped
```

Comment:
I would suggest to add a note that the expected value is to be an array so that in example if you are willing to exclude the space "dev" from the execution you need to provide an array like this one: ["stage"]. 
In case you want to exclude 1+ space, let's say "dev-1" and "dev-2", then the array should be like this one: 
["dev-1","dev-2"]